tests/bsim/bluetooth/host/l2cap/frags/prj.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_BT=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="L2CAP frags test"
+
+CONFIG_BT_EATT=n
+CONFIG_BT_L2CAP_ECRED=n
+
+CONFIG_BT_SMP=y # Next config depends on it
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+
+# Disable auto-initiated procedures so they don't
+# mess with the test's execution.
+CONFIG_BT_AUTO_PHY_UPDATE=n
+CONFIG_BT_AUTO_DATA_LEN_UPDATE=n
+CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=n
```

Comment:
You could argue that if these could affect the execution, then we need to test with them both enabled and disabled. 

I don't think they will mess with this test though, right?

Reply:
I don't want to test the magic. If the test fails, I'd like to be able to not have extra mental overhead. These features will randomly allocate bufs from the common pool and queue them up at the worst times.

I've also seen them crash the controller a bunch of times. I (and @alwa-nordic) think they should be in their own library, and that the application should explicitly call the API instead of this "update my conn, maybe. sometime.".

Reply:
so to answer your last question, yes I added them because they messed up this test.